### ISSUE TITLE
Use enums rather than strings for input

### ISSUE THREAD

**Issue by rozniak** [rozniak/rzxe#9]
Currently inputs are taken in the form of strings such as:
`vk.a`
`vk.s`
`mb.left`
`mb.middle`

It would be better to have this in a kind of enum that covers our inputs, perhaps along the lines of:
`ControlInput.KeyA`
`ControlInput.KeyS`
`ControlInput.Mouse0`
`ControlInput.Mouse3`

This would make the code less error prone due to potential typos in strings (plus strings just seem like a dirty way of doing this kind of thing).

**Comment by TarenLewis** [comment]
I see that you already have the enum MouseButton class, was this the class you were planning to use for this issue?

**Comment by rozniak** [comment]
Hey, thanks for your interest in the project! :grinning: 

`MouseButton` would've been part of my initial ideas - it's from before I set up the repo and uploaded everything in its initial state.

At the moment all the inputs are recorded very simply as collections of new presses/releases (inside `InputEvents`) - as such they'd have to all be part of the same enum. In future it'll need thought about multiple device support (in the case of gamepads) but for now I reckon we can keep it simple.

Essentially bin off `MouseButton`, make a new enum (could be `ControlInput` from my OP, or something better if you have an idea :grin: ) and populate it with keyboard / mouse inputs (mouse 0 - 2... not sure why I wrote 3). Then replace the code that uses strings to instead use the enum.
